coilsnake/assets/modulelist.txt

```diff
@@ -14,6 +14,7 @@ eb.TownMapIconModule
 eb.DeathScreenModule
 eb.SoundStoneModule
 eb.CastModule
+eb.StaffModule
```

Comment:
since this only deals with text, I think it will probably be very quick to compile, so it should go faily high up (I would recommend just below MiscTextModule)

Reply:
Moved there.